homework-submission/todo-app/src/Components/Task.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class Task extends Component {
+    render() {
+        const { todo } = this.props;
+        return (
+            <div className='Task'>
+                <div style={{ textDecoration: todo.done ? 'line-through' : 'none' }}>
```

Comment:
Styling is one of the most important things, But what **If** you wanted to change the styling from your stylesheet file in the future.
This won't change, right? 🙃💔

there are many ways of styling improvements in the react world, Have you a look:
https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822